app/assets/stylesheets/content/_work_packages_table.sass

```diff
@@ -25,41 +25,3 @@
 //
 // See doc/COPYRIGHT.rdoc for more details.
 //++
-
-@import 'table'
```

Comment:
![image](https://cloud.githubusercontent.com/assets/453584/9224720/e6df4c52-4104-11e5-866b-58d908e24849.png)

WHY U NOT DELETE THE WHOLE FILE?